developers/contributing_how_tos.md

```diff
@@ -0,0 +1,41 @@
+# How to contribute a napari How-To
+
+This should be a sentence (or brief paragraph) about what this how-to will achieve and how it helps address a common problem/task/need.
+
+## Prerequisites
+
+- this template
+- a list of what the user needs to prepare to be ready to start this how-to guide 
+- links to any existing napari.org tutorials or how-to guides that can help the user fulfill these prerequisites
+- the level of python/napari knowledge required to follow this how-to
+    - try to be specific about what skills are needed e.g. 
+    'connecting callbacks to layer events' or 'using matplotlib to produce plots'
+- plugins that should be installed
+- python packages that should be installed (don't list napari dependencies)
+
+***
+
+## How-to steps
+
+Headings in this section are yours to decide on. You should split your how-to guide into easy to follow, sequential steps to achieve the stated goal. Remember that your how-to may be used as a reference, so your headings should be descriptive and allow users to get a quick overview of the document, and which steps are relevant to them.
+
+## Include pictures
+
+You should include pictures/videos, particularly when describing interactions with the viewer. If an action can be performed both through the viewer and through code, you should include both options.
+
+## How-to vs. tutorial
+
+A how-to is a step by step guide designed to answer a specific "How do I..." question. This means the user already knows **what** they want to do, just not how. A how-to does not explain concepts or engage in discussion, and should prioritise practical usability. In comparison, a tutorial is a broader exploration, or guided tour of what is available. It might include detailed explanations and discussion, and is usually aimed at users who don't yet know what can be done.
+
+## Where does this how-to go?
+
+Once you've written your how-to, you should place this document under `getting_started/how_tos/` and open a PR in the [napari.github.io](https://github.com/napari/napari.github.io) repository.
```

Comment:
We will probably need to update this to expand a bit more on where in the repository this document could go as I believe there are two sections for how-tos @melissawm?

Reply:
Actually, I didn't anticipate a separate how-to section under the "Getting started" section. But I'm happy to discuss this if you feel it's needed

Reply:
No, I think I just got confused because the getting started section contains some how-tos. But I can see that this section is unlikely to be contributed to, so I guess we direct them to place their new document in a new folder we don't have yet called 'how-to's?